src/zenml/integrations/kubeflow/orchestrators/local_deployment_utils.py

```diff
@@ -9,7 +9,7 @@
 from zenml.logger import get_logger
 from zenml.utils import networking_utils, yaml_utils
 
-KFP_VERSION = "1.7.1"
+KFP_VERSION = "1.8.1"
 # Name of the K3S image to use for the local K3D cluster.
 # The version (e.g. v1.21.9) refers to a specific kubernetes release and is
 # fixed as KFP doesn't support the newest releases immediately.
```

Comment:
Should we create a ticket to update the `K3S_IMAGE_NAME` as well as this newer KFP version probably also supports newer version of kubernetes?

Reply:
Why wait ? Done